activerecord/lib/active_record/connection_adapters/resolver.rb

```diff
@@ -76,12 +81,12 @@ def spec(config)
       #   Resolver.new({}).resolve("postgresql://localhost/foo")
       #   # => DatabaseConfigurations::UrlConfig.new(config: {"adapter" => "postgresql", "host" => "localhost", "database" => "foo"})
       #
-      def resolve(config_or_env, pool_name = nil)
+      def resolve(config_or_env, _pool_name = nil)
```

Comment:
Will we be removing this entirely?

Reply:
Seems like we should at least deprecate the `_pool_name` parameter if we can't remove it yet for backwards compatibility reasons.

Reply:
@dylanahsmith `resolve` is a private API, we don't need to deprecate it.

Reply:
Great.  We should just remove it in that case.

```suggestion
      def resolve(config_or_env)
```

Reply:
I'd like to, but I'm not sure it's an API breakage or not, so I left it.

Reply:
🔥 it. Anything with `:nodoc` or inside a `nodoc` constant can be removed without deprecation.